portfolio.adobe.com.stage.json

```diff
@@ -15,9 +15,9 @@
     },
     {
       "type":"TXT",
-      "host":"@",
-      "data": "%uuid%",
-      "ttl":3600
+      "host":"_portfolio",
+      "data": "verify=%uuid%",
+      "ttl":600
```

Comment:
Erroneously set this to the prod TTL value of 3600 instead of the stage TTL value of 600 when I initially added the TXT record.